test.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:8
+
+WORKDIR /usr/src/app
+
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 8080
+
+CMD [ "npm", "test" ]
```

Comment:
Are you able to say just use `Dockerfile` when you're using `test.docker-compose`? The two files look the same, and that would reduce the code (slightly)!

Reply:
The only difference between the two is:
```
CMD [ "npm", "test" ]
``` 
and 
```
CMD [ "npm", "start" ]
``` 

Unfortunately, the boiler plate that comes before this can't be reduced afaik